--- /usr/lib/python2.7/dist-packages/openerp/addons/web/static/lib/py.js/lib/py.js	2013-06-16 23:26:30.660384152 +0000
+++ /usr/lib/python2.7/dist-packages/openerp/addons/web/static/lib/py.js/lib/py.js	2013-06-16 23:30:02.035589446 +0000
@@ -764,7 +764,14 @@
 
         // Conversion
         toJSON: function () {
-            throw new Error(this.constructor.name + ' can not be converted to JSON');
+            var out = {};
+            for(var k in this) {
+                if (this.hasOwnProperty(k) && !/^__/.test(k)) {
+                    var val = this[k];
+                    out[k] = val.toJSON ? val.toJSON() : val;
+                }
+            }
+            return out;
         }
     });
     var NoneType = py.type('NoneType', null, {
